pubspec.yaml

```diff
@@ -32,8 +32,14 @@ dependencies:
   hive: ^2.0.0
   hive_flutter: ^1.0.0
   lint: ^1.0.0
-  path_provider: ^2.0.1
-  google_fonts: ^2.0.0-nullsafety.0
+  ionicons: ^0.1.2
+
+
+
+  # The following adds the Cupertino Icons font to your application.
+  # Use with the CupertinoIcons class for iOS style icons.
+  cupertino_icons: ^1.0.0
```

Comment:
Kindly remove this line "  cupertino_icons: ^1.0.0" . This package has already been included on top.